### ISSUE TITLE
Expected Accuracy

### ISSUE THREAD

**Issue by ericleonardo** [Atskyr/Keras-tensorflow-porfolio-development-AI#1]
Hi! Please, what is the average directional accuracy we can expect?
Did you get 60% accuracy for binary up/down prediction?

Thanks for sharing and congratulations for the great work!
